packages/IMAPClient-Core.package/ICAddressBook.class/instance/feedMeWithEntries..st

```diff
@@ -0,0 +1,7 @@
+initialize
+feedMeWithEntries: anEmailCollection
```

Comment:
maybe think about that method name... fillWithEntries or seedWithEntries feels more fitting.
(Actually just the "me" in the name is disturbing)

Reply:
I wouldn't agree since it actually describes exactly what this method's doing. 
If you call `addressbook feedMeWithEntries` the addressbook itself starts collecting addresses and fills itself with it.